windows-msi/version.m4

```diff
@@ -42,15 +42,15 @@ define([PRODUCT_NAME],      [OpenVPN])
 define([PRODUCT_PUBLISHER], [OpenVPN, Inc.])
 
 dnl The package version as displayed by UI and used in filenames (no spaces, please).
-define([PACKAGE_VERSION], [2.7git])
+define([PACKAGE_VERSION], [2.7_git-I001])
```

Comment:
is this and (and below) an unrelated change?

Reply:
Indeed.

Reply:
Yes. When testing the build locally the tools updated this. I left it in although it makes no real difference, since it makes sense to me having the PACKAGE_VERSION closer to what we actually use.